### DISCUSSION TITLE
Is code for building installers available?

### DISCUSSION THREAD

**Discussion by pplonski** [mindsdb/mindsdb#1243] [Get help] [answered]
Is code for building installers available somewhere? I would like to check how the macOS installer is created.

**Comment by ZoranPandovski** [comment, marked as answer]
Hello @pplonski, atm no. It is on the private repo. Do you experience some issues while trying to use it? Maybe I can share that with you

**Reply by pplonski** [reply]
I didn't experience any issues. It works very smoothly. I found a Data Science project called nteract that doesn't have such an all-in-one installer. I even add an issue at nteract repo for the all-in-one installer https://github.com/nteract/nteract/discussions/5535

I just want to check how to make such an installer.

**Reply by ZoranPandovski** [reply]
I haven't used nteract but looks interesting. About the actual installers code, I will not be able to shared atm. But to give you more info we have built it using the pyinstaller to build the executables https://www.pyinstaller.org/

**Reply by pplonski** [reply]
Thank you for your help @ZoranPandovski!
